scripts/mbedtls_dev/bignum_core.py

```diff
@@ -91,25 +91,26 @@ def generate_function_tests(cls) -> Iterator[test_case.TestCase]:
             yield cls(a_value, b_value, 32).create_test_case()
             yield cls(a_value, b_value, 64).create_test_case()
 
-class BignumCoreAddIf(BignumCoreOperation):
+class BignumCoreAddIf(BignumCoreOperationArchSplit):
```

Comment:
If you've got an `ArchSplit` can you add something with inheritance to call the correct `bound_mpiX()` based on just calling `bound_mpi()`?

Reply:
Yes, I think that it is a good idea. In the followup I have something similar with formatting, we could do the same for the bound.

Reply:
Actually this commit does it already:
https://github.com/Mbed-TLS/mbedtls/pull/6446/commits/09a298570160af9fb6288531eff7ff6359c9a049